### ISSUE TITLE
Error Showing in integrating the Khalti Payment Gateway in opencart

### ISSUE THREAD

**Issue by cnswebtech** [khalti/khalti-opencart#1]
Follow all the steps to install the khalti gateway in opencart 3.0.2.0. but it shows the error.

![khalti-error](https://user-images.githubusercontent.com/41678281/43194313-b001f820-901f-11e8-99d7-0e47c9b5bf41.jpg)

**Comment by manibibek** [comment]
@cnswebtech this issue has been fixed now, can you please check

Thanks

**Comment by cnswebtech** [comment]
No , yet this is showing error 

![ntface-1](https://user-images.githubusercontent.com/41678281/43198427-3fbdfc5a-902c-11e8-94d9-1349bd5688c5.png)

**Comment by manibibek** [comment]
@cnswebtech did you download the latest code after I made the changes?

**Comment by cnswebtech** [comment]
yes , I download the fresh code and upload that on the server. but still same error.

**Comment by manibibek** [comment]
That shouldn't be the case, as you can see there is no any validate() function being called on line 18 of that file on the latest commit. 
Please see the commit changes below.
https://github.com/khalti/khalti-opencart/commit/8848f1a89794c8153bce8b974c1ebd3aec16ab4c
